docs/blog/2019-11-22-getting-started-with-gatsby-cloud/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Making it easier to start with Gatsby Cloud for free
+date: 2019-11-2
+author: Shannon Soper
+tags: ["ux", "cloud", "cms"]
+---
+
+When you go to [our Quickstart page](https://www.gatsbyjs.com/get-started/) to try Gatsby Cloud, you’ll probably find yourself in one of two situations.
+
+Situation 1: you already have a Gatsby project
+Situation 2: you don’t have a Gatsby project yet
+
+![A screenshot of the current Get Started page on gatsbyjs.com](final-state.png)
+
+Either way, Gatsby Cloud should work for you! Through the [Cloud Quickstart page]( https://www.gatsbyjs.com/get-started/), you can integrate your site with its data source(s) to drive Preview and automate builds. And it’s no commitment and no credit card required. All you need is a Github account.
```

Comment:
```suggestion
Either way, Gatsby Cloud will work for you! Through the [Cloud Quickstart page]( https://www.gatsbyjs.com/get-started/), you can integrate your site with its data source(s) to drive Preview and automate builds. And it’s no commitment and no credit card required. All you need is a Github account.
```

Maybe will instead of should?